Altis_Life.Altis/core/admin/fn_adminCompensate.sqf

```diff
@@ -8,9 +8,10 @@
 */
 private ["_value","_action"];
 if (FETCH_CONST(life_adminlevel) < 2) exitWith {closeDialog 0; hint localize "STR_ANOTF_ErrorLevel";};
-_value = parseNumber(ctrlText 9922);
-if (_value < 0) exitWith {};
-if (_value > 999999) exitWith {hint localize "STR_ANOTF_Fail"};
+_value = ctrlText 9922;
+if (!([_value] call life_fnc_isNumber)) exitWith {hint localize "STR_ATM_notnumeric"};
+_value = parseNumber(_value);
+if (_value < 1) exitWith {};
```

Comment:
We can't do that, because for example:
45A5B2C will result a number greater than 0.
Also, isNumber prevents fractions. (Ex: 4502.5)